EntranceShuffle.py

```diff
@@ -85,7 +85,7 @@ def build_one_way_targets(world, types_to_include, exclude=(), target_region_nam
                         ('Jabu Jabus Belly Beginning -> Zoras Fountain',                    { 'index': 0x0221, 'blue_warp': 0x010E })),
     ('Dungeon',         ('SFM Forest Temple Entrance Ledge -> Forest Temple Lobby',         { 'index': 0x0169 }),
                         ('Forest Temple Lobby -> SFM Forest Temple Entrance Ledge',         { 'index': 0x0215, 'blue_warp': 0x0608 })),
-    ('Dungeon',         ('DMC Fire Temple Entrance -> Fire Temple Lower',                   { 'index': 0x0165 }),
+    ('Dungeon',         ('DMC Fire Temple Exit -> Fire Temple Lower',                       { 'index': 0x0165 }),
```

Comment:
I found these new names confusing because the prior name meant "the Fire Temple Entrance in DMC", and now this is not "the Fire Temple exit" (which I was then expecting to be a region with "DMC: True" but it has "Fire Temple: True"); it's more like "the DMC exit to Fire Temple"... I'll suggest "DMC Exit to ..." for all of these.